mockiato_codegen/src/generate/mock_struct.rs

```diff
@@ -124,8 +134,17 @@ fn generate_expect_method(
         TokenStream::new()
     };
 
+    let documentation = LitStr::new(
+        &format!(
+            "Expects a call to [`{0}::{1}`].\n\n[`{0}::{1}`]: ./trait.{0}.html#tymethod.{1}",
```

Comment:
"..., panicking if the function was not called by the time the object goes out of scope"